Formula/libgsm.rb

```diff
@@ -31,8 +31,10 @@ def install
     man1.mkpath
     man3.mkpath
 
+    inreplace "Makefile", "libgsm.1.0.13.dylib", "libgsm.1.0.16.dylib"
```

Comment:
>And Upstream said that they won't provide dylib.

Do you have a reference for this? That may be sufficient reason to remove this from Homebrew.

>Original Makefile by upstream don't have string of "libgsm.1.0.13.dylib".
It was written by the Patch.

The patch should be updated accordingly and submitted to https://github.com/Homebrew/formula-patches.